app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html

```diff
@@ -81,6 +82,21 @@ <h3 class="govuk-heading-m">@messages("inspection_needed_export.arriving.h3")</h
         <li><a href="@{appConfig.checkDelaysInlandBorderFacilityUrl}">@messages("inspection_needed_export.arriving.p1.link2")</a></li>
     </ul>
 
+    @govukInsetText(InsetText(
```

Comment:
are you not checking for direction check here ? if so it will show the content for all 4 directions?

Reply:
this particular view is just for GB to EU, I think

Reply:
Done